skipper/cli.py

```diff
@@ -36,12 +37,17 @@ def cli(ctx, registry, build_container_image, build_container_tag, build_contain
     ctx.obj['build_container_net'] = build_container_net
     ctx.obj['git_revision'] = build_container_tag == 'git:revision'
     ctx.obj['build_container_tag'] = git.get_hash() if ctx.obj['git_revision'] else build_container_tag
-    ctx.obj['env'] = ctx.default_map.get('env', {})
     ctx.obj['containers'] = ctx.default_map.get('containers')
     ctx.obj['volumes'] = ctx.default_map.get('volumes')
     ctx.obj['workdir'] = ctx.default_map.get('workdir')
     ctx.obj['workspace'] = ctx.default_map.get('workspace', None)
     ctx.obj['container_context'] = ctx.default_map.get('container_context')
+
+    env = ctx.default_map.get('env', {})
+    if env_file:
+        env.update(utils.load_yaml_file(env_file))
```

Comment:
Seems that the env file will override the env in the cmd, it should be the other way around.

Reply:
I think a better implementation will be to pass the env file forward (with --env-file) to doker/podman and let it handle it.

Reply:
> I think a better implementation will be to pass the env file forward (with --env-file) to doker/podman and let it handle it.

done, opened https://github.com/Stratoscale/skipper/pull/139